src/libraries/Microsoft.Bcl.Numerics/src/Microsoft.Bcl.Numerics.csproj

```
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;netstandard2.1;$(NetFrameworkMinimum)</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+
+    <!-- This assembly should never be placed inbox as it is only for downlevel compatibility. -->
+    <PackageDescription>Provides the System.MathF for .NET Standard 2.0</PackageDescription>
+        <!-- Disabling baseline validation since this is a brand new package.
+       Once this package has shipped a stable version, the following line
+       should be removed in order to re-enable validation. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OmitResources Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">true</OmitResources>
```

That doesn't work as the TargetFrameworkIdentifier property isn't available as a property at this point of time. You will need to use this instead:

```suggestion
    <OmitResources Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</OmitResources>
```